packages/dina-ui/components/collection/material-sample/organismUtils.ts

```diff
@@ -50,3 +51,23 @@ function getDeterminations(organisms: Organism[]) {
 
   return determinationList.join(", ");
 }
+
+export function getMaterialSampleSummaryScientificNames(
+  determinations: Determination[] | undefined
+): string {
+  if (!determinations) {
+    return "";
+  }
+  const scientificNames: string[] = [];
+  determinations.forEach((determination) => {
+    if (determination.isPrimary) {
+      if (determination.scientificName) {
+        scientificNames.push(determination.scientificName);
+      } else if (determination.verbatimScientificName) {
+        scientificNames.push(determination.verbatimScientificName);
+      }
+    }
+  });
+
+  return scientificNames.join(", ");
```

Comment:
The `getDeterminations` function should be altered to take determinations instead of organisms (the `getScientificNames` method would also need to be altered a bit so it returns the determinations of that organism instead of passing the organism directly)

The `getDeterminations` and `getMaterialSampleSummaryScientificNames` seem to be very similar so we might be able to remove some code duplication here.